[docs] Add vividus dependency maintain rule (#4293)
vividus-framework/vividus: docs/modules/ROOT/partials/dependency-installation.adoc

. Copy the below line to `dependencies` section of the project `build.gradle` file
+
[IMPORTANT]

> Please make sure the VIVIDUS version is the same across all VIVIDUS dependencies in your `build.gradle` file whether you add a new dependency or update an existing one.

Reviewer: ```suggestion
Please make sure to use the same version for all VIVIDUS dependencies.
```
